src/Strategy/Node/Errors.hs

```diff
@@ -13,12 +15,19 @@ import Prettyprinter (Pretty (pretty), indent, vsep)
 yarnLockfileDocUrl :: Text
 yarnLockfileDocUrl = "https://classic.yarnpkg.com/lang/en/docs/yarn-lock/"
 
+yarnV2LockfileDocUrl :: Text
+yarnV2LockfileDocUrl = "https://yarnpkg.com/getting-started/qa#should-lockfiles-be-committed-to-the-repository"
+
 npmLockFileDocUrl :: Text
 npmLockFileDocUrl = "https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json"
 
 fossaNodeDocUrl :: Text
 fossaNodeDocUrl = strategyLangDocUrl "nodejs/nodejs.md"
 
+data CyclicPackageJson = CyclicPackageJson
+instance ToDiagnostic CyclicPackageJson where
+  renderDiagnostic (CyclicPackageJson) = "We detected cyclic references in package.json"
```

Comment:
Technically this is inaccurate as written:
How about: `We detected cyclic references between package.json files in the workspace.`